site/docs/operations/cdc.md

```diff
@@ -312,3 +312,91 @@ of a standby.
 There are discussions in the PostgreSQL community around a feature
 called *failover slots* which would help mitigate this problem, but as
 of version 12 they have not been implemented yet.
+
+## Troubleshooting
+
+### MySQL
+
+The general behaviour of the MySQL connector when loosing connection to
+the database is governed by a configurable reconnect strategy and a
+boolean flag specifying if state should be reset on reconnects or not.
+For details see the
+[javadoc](/javadoc/{jet-version}/com/hazelcast/jet/cdc/mysql/MySqlCdcSources.html).
+
+There are however some discrepancies and peculiarities in the behavior.
+
+#### During snapshotting
+
+If the connection to the database fails during the snapshotting phase
```

Comment:
We should be rather more precise here and instead of `snapshotting` use `database snapshotting` or something like that otherwise it is not clear whether we talk about db snapshotting of jet snapshotting.